diff --git a/tesseract_motion_planners/CMakeLists.txt b/tesseract_motion_planners/CMakeLists.txt
index 14eb54a..e2d9a86 100644
--- a/tesseract_motion_planners/CMakeLists.txt
+++ b/tesseract_motion_planners/CMakeLists.txt
@@ -74,7 +74,7 @@ if(TESSERACT_BUILD_TRAJOPT_IFOPT)
 endif()
 
 # Examples
-add_subdirectory(examples)
+# add_subdirectory(examples)
 
 # Tests
 if(TESSERACT_ENABLE_TESTING)
diff --git a/tesseract_process_managers/CMakeLists.txt b/tesseract_process_managers/CMakeLists.txt
index 779066b..c438c27 100644
--- a/tesseract_process_managers/CMakeLists.txt
+++ b/tesseract_process_managers/CMakeLists.txt
@@ -10,6 +10,7 @@ if(WIN32)
 endif()
 
 # System dependencies are found with CMake's conventions
+find_package(jsoncpp REQUIRED)
 find_package(Eigen3 REQUIRED)
 find_package(Taskflow REQUIRED)
 find_package(console_bridge REQUIRED)
@@ -132,7 +133,7 @@ target_include_directories(${PROJECT_NAME}_utils PUBLIC "$<BUILD_INTERFACE:${CMA
                                                         "$<INSTALL_INTERFACE:include>")
 list(APPEND Targets ${PROJECT_NAME}_utils)
 
-add_subdirectory(examples)
+# add_subdirectory(examples)
 
 configure_package(NAMESPACE tesseract TARGETS ${Targets})
 
